mscore/inspector/inspector.cpp

```diff
@@ -501,6 +503,25 @@ InspectorStaffTypeChange::InspectorStaffTypeChange(QWidget* parent)
       mapSignals();
       }
 
+//---------------------------------------------------------
+//   setElement
+//---------------------------------------------------------
+
+void InspectorStaffTypeChange::setElement()
+      {
+      InspectorBase::setElement();
+      for (Element* el : *(inspector->el())) {
+            StaffTypeChange* stc = toStaffTypeChange(el);
+            // tab staff shouldn't have key signature
+            if (stc->staffType()->group() == StaffGroup::TAB) {
+                  sl.genKeysig->setChecked(false);
```

Comment:
This may cause key signatures disappear on a normal staff type change. To reproduce try the following:

1) Open/create a score with normal and tablature staves, add a key signature;
2) Add staff type changes to both normal and tablature staves;
3) Select staff type change on a normal staff;
4) Hold Ctrl and select staff type change on a tablature staff.

Result: key signatures disappear on a normal staff after the staff type change element.

Reply:
Fixed.